lib/datadog/tracing/trace_operation.rb

```diff
@@ -373,6 +375,14 @@ def fork_clone
         )
       end
 
+      # When APM is running in 'non-billing' mode, we should skip distributed tracing
+      # unless specific distributed tags are present. These tags can come from upstream
+      # services, or from the service itself.
+      # For now, only '_dd.p.appsec=1' is supported.
+      def non_billing_reject?
```

Comment:
This method name contains two negatives. I would prefer something like `should_send?` for example but before renaming let's figure out the overall name for the setting.